.mvn/jvm.config

```diff
@@ -0,0 +1,10 @@
+--add-exports jdk.compiler/com.sun.tools.javac.api=ALL-UNNAMED
```

Comment:
Question: What is all this?

Reply:
This is part of the installation guide for Maven: https://errorprone.info/docs/installation